flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -189,6 +195,8 @@
 
     private final List<JobListener> jobListeners = new ArrayList<>();
 
+    private final Map<String, ResourceProfile> slotSharingGroupResources = new HashMap<>();
```

Comment:
Maybe a "default" group with `UNKNOWN` resources should be registered by default?

Reply:
`slotSharingGroupResources` should only contain specified resources, I'll claim this in the javadoc.